cmd/metal-api/internal/metal/provisioning.go

```diff
@@ -4,9 +4,56 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/looplab/fsm"
 	"go.uber.org/zap"
 )
 
+// FSM states
+const (
+	FsmStateInitial          string = "Initial State"
```

Comment:
Not sure why copying the states is necessary when they are already defined below?